src/Compilers/Core/Portable/Operations/OperationExtensions.cs

```diff
@@ -79,23 +105,12 @@ public static ImmutableArray<ILocalSymbol> GetDeclaredVariables(this IVariableDe
             return arrayBuilder.ToImmutableAndFree();
         }
 
-        private sealed class OperationCollector : OperationWalker
+        /// <summary>
+        /// Deep Clone given IOperation
+        /// </summary>
+        internal static T Clone<T>(this T operation) where T : IOperation
```

Comment:
As far as I can tell, Clone is only used in tests. Could you share a scenario where it will be useful in our codebase?